qCC/ccCommandLineCommands.cpp

```diff
@@ -165,6 +165,50 @@ constexpr char OPTION_FILE_NAMES[]						= "FILE";
 constexpr char OPTION_ORIENT[]							= "ORIENT";
 constexpr char OPTION_MODEL[]							= "MODEL";
 
+bool get_SFIndex_SFName(ccCommandLineInterface& cmd, int& sfIndex, QString& sfName)
```

Comment:
To remain consistent with the project coding style, I would name it:
_static bool GetSFIndexOrName(...)_